internal/action/test_utils.go

```diff
@@ -0,0 +1,34 @@
+package action
```

Comment:
Always `_test.go` for test build content

https://pkg.go.dev/cmd/go/internal/test